Main.cpp

```diff
@@ -140,20 +140,22 @@ enum StarSize {
 
 struct Star {
     // Bitmap representation
-    BITMAP *m_pBitmap;
+    BITMAP *m_pBitmap = nullptr;
     // Center location on screen
-    Vector m_Pos;
-    // Bitmap offset
-//    int m_Offset;
+	std::array<int, 2> m_Pos = { 0 };
```

Comment:
I don't love using the std array for this. I get that you picked it over Vector cause it's lightweight and doesn't use floats, but it feels really odd. I think my preference would actually be put together a new `starPosition` struct or something, with X and Y values.
Alternatively, you could separate the X and Y values into two variables in the struct, I'd be fine with that too (this is actually probably the best option now I think more).
If you disagree, we can discuss, this is just my feeling on the matter, there's nothing really wrong with how you've done it.

Reply:
I like the std array but I don't mind changing to X and Y variables.
I'm not even using array methods so I guess it's functionally the same.